lib/webpacker/compiler.rb

```diff
@@ -89,7 +89,7 @@ def run_webpack
         end
       else
         non_empty_streams = [stdout, stderr].delete_if(&:empty?)
-        logger.error "Compilation failed:\n#{non_empty_streams.join("\n\n")}"
+        logger.error "Compilation failed:\n ExitStatus: #{status}\n Outputs: #{non_empty_streams.join("\n\n")}"
```

Comment:
I think we can make this even a bit more obvious. 

Can you try adding a `\n` before the non_empty_streams part, as your example shows it's hard to see how the messages are grouped by the indenting?

Reply:
We can add a blank line before this and make all caps.

> Compilation failed:

Reply:
@justin808 Thanks for the comment.
The following is the result of the modification ( 5900430 ).
Is this output consistent with your assumptions?

```
** Execute webpacker:compile
Compiling...

COMPILATION FAILED:
 EXIT STATUS: pid 244 SIGKILL (signal 9)
 OUTPUTS:
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
```